dawn/src/dawn/CodeGen/Options.inc

```diff
@@ -32,6 +32,11 @@
 //                  command-line target will be generated which allows to negate the option i.e
 //
 
+// TODO Rename this to (MaxHaloPoints, "max-halo") after combined struct is eliminated
```

Comment:
This TODO is somewhat important and will be eventually removed, so I vote we keep it for now.

Reply:
:+1: